qiskit_ionq/ionq_client.py

```diff
@@ -39,7 +39,7 @@ class IonQClient:
     """IonQ API Client
 
     Attributes:
-        _url(str): A URL base to use for API calls, e.g. ``"https://api.ionq.co/v0.1"``
```

Comment:
We probably want to do a minor or major semver bump in qiskit

I think we'll need to say retrieve_job no longer populates histogram

Reply:
Oh, actually, I don't think we expose that class (nice)

Reply:
https://qiskit.org/documentation/partners/ionq/apidocs/backends.html ah, It's in backend.

Reply:
All good, we expose sane wrappers in all cases I can see on https://qiskit.org/documentation/partners/ionq/apidocs/backends.html